code/game/machinery/pipe/construction.dm

```diff
@@ -611,6 +785,52 @@
 				M.node3.atmos_init()
 				M.node3.build_network()
 
+		if(PIPE_FUEL_MANIFOLD)		//manifold
+			var/obj/machinery/atmospherics/pipe/manifold/hidden/fuel/M = new( src.loc )
+			M.color = color
+			M.set_dir(dir)
+			M.initialize_directions = pipe_dir
+			//M.New()
+			var/turf/T = M.loc
+			M.level = !T.is_plating() ? 2 : 1
+			M.atmos_init()
+			if (!M)
+				to_chat(usr, "There's nothing to connect this manifold to! (with how the pipe code works, at least one end needs to be connected to something, otherwise the game deletes the segment)")
```

Comment:
```suggestion
				to_chat(usr, "There's nothing to connect this manifold to!")
```

Reply:
This is present in -all- the manifolds. I object to changing only these two pipes.